runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPointsOverlay.java

```diff
@@ -80,6 +83,42 @@ public Dimension render(Graphics2D graphics)
 			.right(String.valueOf(client.getVar(Varbits.RAID_PARTY_SIZE)))
 			.build());
 
+		if (config.uniqueChance())
+		{
+			double chance = Math.min(totalPoints / 7125.0, 80.0);
```

Comment:
Just calculate double percent = totalPoints / 7125.0 once and replace 570000.0 with 80 and 1140000.0 with 160.

Reply:
Also put the double values such as 7125.0 and 80.0 into constants to make sense of their meaning.

Reply:
Yeah exactly, that would look a lot cleaner. Just make sure not to have any magic numbers.